src/PhpSpec/Console/ResultConverter.php

```diff
@@ -21,6 +21,23 @@
 class ResultConverter
 {
     /**
+     * @var IO
+     */
+    private $io;
+
+
+    /**
+     * ResultConverter constructor.
+     *
+     * @param IO $io
+     */
+    public function __construct(IO $io)
+    {
+        $this->io = $io;
+    }
+
+
```

Comment:
It's at a different point in the file ;-) In general single line between methods please